lib/preprocessors/glimmer.js

```diff
@@ -22,11 +17,16 @@ function arrayEq(arr1, arr2) {
 const oneLineTemplateRegex = /\[__GLIMMER_TEMPLATE\(`(.*)`, {.*}\)]/;
 
 // regex to capture opening template tag
-const openingTemplateTagRegex = /\[__GLIMMER_TEMPLATE\(`$/;
+const openingTemplateTagRegex = /\[__GLIMMER_TEMPLATE\(`/;
 
 // regex to capture closing template tag
 const closingTemplateTagRegex = /`, {.*}\)]/;
 
+// regex with capture group on scope token variables. In the following example:
+// `, { strictMode: true, scope: () => ({on,noop}) })]
```

Comment:
are these always on a single line?, even for very large scope objects?

Reply:
from what I have seen, these transforms will always be on the same line. We might be able to confirm this by pulling on the replacements here: https://github.com/ember-template-imports/ember-template-imports/blob/f273bd21c7bbd310bb75c928a5cc06225ecbdf8f/src/preprocess-embedded-templates.ts#L133



<img width="1850" alt="image" src="https://user-images.githubusercontent.com/6970731/224833256-f44213c0-22e6-45d9-9e91-0c621c7e9d7c.png">